color/src/missing.rs

```diff
@@ -3,45 +3,49 @@
 
 //! A simple bitset.
 
-/// A simple bitset, for representing missing components.
+/// A simple bitset for representing missing components.
 #[derive(Default, Clone, Copy, PartialEq, Eq, Debug)]
-pub struct Bitset(u8);
+pub struct Missing(u8);
 
-impl Bitset {
+impl Missing {
+    /// Returns `true` if the set contains the component index.
     pub fn contains(self, ix: usize) -> bool {
         (self.0 & (1 << ix)) != 0
     }
 
-    pub fn set(&mut self, ix: usize) {
+    /// Adds a component index to the set.
+    pub fn insert(&mut self, ix: usize) {
         self.0 |= 1 << ix;
     }
 
-    pub fn single(ix: usize) -> Self {
+    /// The set containing a single component index.
+    pub fn singleton(ix: usize) -> Self {
```

Comment:
`singleton` seems like a confusing name for this. This usually implies what in Rust would be a static instance. I would prefer the old name of `single`. Or perhaps an idiomatic rusty name would be `with_single_component`.

Reply:
I've gone back to `single`, at your suggestion.